client/src/components/table/Table.js

```diff
@@ -6,104 +6,152 @@ import 'ag-grid-community/dist/styles/ag-grid.css';
 import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
 
 const Table = () => {
+  // Sort Birth Dates in ascending order (earlier dates first)
+  const dateComparator = (value1, value2) => {
+    const date1 = new Date(value1);
+    const date2 = new Date(value2);
+    return date1 - date2;
+  };
+
+  // Sort Grad Year && Generation in ascending order (earlier cohorts first)
+  const semesterComparator = (value1, value2) => {
+    const sem1 = value1.split(' ')[0];
+    const year1 = value1.split(' ')[1];
+    const sem2 = value2.split(' ')[0];
+    const year2 = value2.split(' ')[1];
+    if (year1 < year2) {
+      return -1;
+    } else if (year1 > year2) {
+      return 1;
+    } else {
+      if (sem1 === 'Fall' && sem2 === 'Spring') {
+        return 1;
+      } else if (sem1 === 'Spring' && sem2 === 'Fall') {
+        return -1;
+      } else {
+        return 0;
```

Comment:
A little nit-picky, but you could remove all these nested statements since you're returning early. So

```
  if (year1 < year2)
      return -1;
  if (year1 > year2)
      return 1;
  if (sem1 === 'Fall' && sem2 === 'Spring')
        return 1;
  if (sem1 === 'Spring' && sem2 === 'Fall')
        return -1;
  return 0;
```

Reply:
Ah yes that's much cleaner, will fix that!